horreum-backend/src/test/java/io/hyperfoil/tools/horreum/svc/RunServiceTest.java

```diff
@@ -58,32 +66,222 @@ public class RunServiceTest extends BaseServiceTest {
    @Inject
    EventBus eventBus;
 
+   private static final int POLL_DURATION_SECONDS = 10;
+
+   @org.junit.jupiter.api.Test
+   public void testStraightforwardTransformationEmptyResult(TestInfo info) throws InterruptedException {
```

Comment:
For consistency I would call this `testTransformationNoSchemaInData` or something similar.

Reply:
changed.